examples/src/main/java/io/druid/examples/function/GuavaDoubleMath.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.examples.function;
+
+import com.google.common.math.DoubleMath;
+import io.druid.math.expr.BuiltinFunctions.SingleParam;
+import io.druid.math.expr.BuiltinFunctions.TripleParam;
+import io.druid.math.expr.ExprEval;
+import io.druid.math.expr.Function;
+
+/**
+ */
+public class GuavaDoubleMath implements Function.Library
```

Comment:
hm, I'm okay with calling this "Guava", to make it clear where the function implementations come from